### ISSUE TITLE
Fabric Hover card doesn't have accessibility support

### ISSUE THREAD

**Issue by sampascal** [OfficeDev/office-ui-fabric-react#11085]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

#### Actual behavior:

The control is not focusable

#### Expected behavior:

The control should be focusable

**Comment by JasonGore** [comment]
The [HoverCard documentation](http://localhost:4322/#/examples/hovercard) covers accessibility considerations and provides an example. I am able to focus and navigation inside of the HoverCard as described by Example 2.

**Comment by sampascal** [comment]
"When using the keyboard to tab to it, open the card with the hotKey and it will automatically focus the first focusable element in the card allowing further navigation inside the card."

What is the hotKey?

**Comment by JasonGore** [comment]
It's a prop value. In this case, the example has it configured as enter. If I tab to a row, hit the right arrow key to "hover" over the key, and then hit "enter", focus enters the HoverCard. I can update the docs to make this clearer.
